code/game/machinery/suit_storage_unit.dm

```diff
@@ -32,9 +32,8 @@
 	mask_type = /obj/item/clothing/mask/breath
 
 /obj/machinery/suit_storage_unit/captain
-	suit_type = /obj/item/clothing/suit/space/captain
-	helmet_type = /obj/item/clothing/head/helmet/space/captain
-	mask_type = /obj/item/clothing/mask/gas
+	suit_type = /obj/item/clothing/suit/space/hardsuit/captain
+	mask_type = /obj/item/clothing/mask/gas/sechailer

```

Comment:
The sprite actually goes well with either. 

That part's mostly an aesthetic detail, I guess. Think I should drop that out?